### DISCUSSION TITLE
Domain being rewritten and I don't know why

### DISCUSSION THREAD

**Discussion by neilio** [AdguardTeam/AdGuardHome#2767] [Q&A]
Hi all -

I've been using AdGuard Home for just over a year and it's worked well for my needs. One issue I've run into on occasion is a domain won't resolve. When I look at the logs, it says the domain was rewritten to 127.0.0.1 but I don't have a rule for this and can't figure out why it's doing this.

Does anyone have any suggestions where I can look to track down what's going on?

Running on Mac OS 10.15 on an old mac mini, version 0.105-1 beta.

**Comment by ameshkov** [comment]
What domain is it?

Could it be that you have it in `/etc/hosts`?
